packages/debug/src/browser/debug-configuration-manager.ts

```diff
@@ -189,7 +192,21 @@ export class DebugConfigurationManager {
         }
     }
     async addConfiguration(): Promise<void> {
-        const { model } = this;
+        let model: DebugConfigurationModel | undefined;
+        if (this.models.size > 1) {
+            const rootUris = (await this.workspaceService.roots).map(r => r.uri);
+            model = await this.quickPick.show(rootUris.map(rootUri => {
+                const modelValue = this.models.get(rootUri);
+                return {
+                    label: this.labelProvider.getName(new URI(rootUri)),
```

Comment:
I think we can align with the 'new terminal' and also display the path to the workspace (helps when there are multiple roots with the same name):

https://github.com/eclipse-theia/theia/blob/c7182b5ab13e6654852c5c5b4acdd6306f6d04fc/packages/terminal/src/browser/terminal-frontend-contribution.ts#L568-L571

Reply:
Your feedback has been resolved in the lastest patch. Thank you !